transport/address.go

```diff
@@ -32,6 +32,7 @@ type Address interface {
 	Listen(name string, i *identity.TokenId, incoming chan Connection, tcfg Configuration) (io.Closer, error)
 	MustListen(name string, i *identity.TokenId, incoming chan Connection, tcfg Configuration) io.Closer
 	String() string
+	BindableAddress() string
```

Comment:
Why did we need to make this a part of the interface?

Reply:
We didn't actually need to.  I will remove.  Stand by.

Reply:
fixed